hq/markdown/wazuh.md

```diff
@@ -154,7 +154,18 @@ Feb 15 11:20:21 ip-10-248-48-147 authenticate-with-wazuh-manager.sh[1068]: 2021/
 STDERR:
 ```
 
-And that's it, all done! On to the next app. 
+And that's it, all done! On to the next app.
+
+If the correct permissions aren't available on the instance, you'll see permissions errors in the output – check that the permissions described above apply to the instance.
+
+```
+ Active: active (running) since Thu 2021-04-08 07:55:29 UTC; 15min ago
+Apr 08 07:55:19 ip-172-31-9-33 authenticate-with-wazuh-manager.sh[1084]: An error occurred (AccessDenied) when calling the DescribeAutoScalingInstances operation: User: arn:aws:sts::563563610310:assumed-role/media-service-TEST-UsageRole-1S2DRMKVLR4UJ/i-0548e6681a83c0a95 is not authorized to perform: autoscaling:DescribeAutoScalingInstances
+Apr 08 07:55:20 ip-172-31-9-33 authenticate-with-wazuh-manager.sh[1084]: An error occurred (AccessDenied) when calling the DescribeAutoScalingGroups operation: User: arn:aws:sts::563563610310:assumed-role/media-service-TEST-UsageRole-1S2DRMKVLR4UJ/i-0548e6681a83c0a95 is not authorized to perform: autoscaling:DescribeAutoScalingGroups
+Apr 08 07:55:21 ip-172-31-9-33 authenticate-with-wazuh-manager.sh[1084]: An error occurred (AccessDenied) when calling the DescribeAutoScalingGroups operation: User: arn:aws:sts::563563610310:assumed-role/media-service-TEST-UsageRole-1S2DRMKVLR4UJ/i-0548e6681a83c0a95 is not authorized to perform: autoscaling:DescribeAutoScalingGroups
+Apr 08 07:55:21 ip-172-31-9-33 authenticate-with-wazuh-manager.sh[1084]: An error occurred (AccessDenied) when calling the DescribeAutoScalingGroups operation: User: arn:aws:sts::563563610310:assumed-role/media-service-TEST-UsageRole-1S2DRMKVLR4UJ/i-0548e6681a83c0a95 is not authorized to perform: autoscaling:DescribeAutoScalingGroups
+Apr 08 07:55:21 ip-172-31-9-33 authenticate-with-wazuh-manager.sh[1084]: 2021/04/08 07:55:21 agent-auth: INFO: Valid key created. Finished.
+```
```

Comment:
Would be good to avoid including the account IDs as this is a public repo.
Could we switch to using example role ARNs and instance IDs or redacting parts?

```
arn:aws:sts::112233445566:assumed-role/instance-role/i-a00b11c22
```

Reply:
_> raises PR in security HQ_
_>has security problems_

😅  PR incoming

Reply:
https://github.com/guardian/security-hq/pull/222 👍